### DISCUSSION TITLE
simulating a larger trafo

### DISCUSSION THREAD

**Discussion by willemijnbrus** [PowerGridModel/.github#37] [Q&A] [answered]
Hi powergridmodel team,

I would like to simulate a larger transformer in my grid to see if a larger transformer would be enougth to solve the problems in the grid. 
The current transformer is 400kVA, I would like to replace it with one of 630kVA. However, I am not sure if I can simply change the Sn or that I need to take into account other dependent variables. When I read my grid I assign values to:
array([(65423, 11561964, 11560899, 1, 1, 10750., 420., 400000., 0.04, 3750., 0.00223002, 515., 2, 1, 5, 0, 2, 5, 1, -128, 250., nan, nan, nan, nan, nan, nan, nan, nan)],
      dtype={'names': ['id', 'from_node', 'to_node', 'from_status', 'to_status', 'u1', 'u2', 'sn', 'uk', 'pk', 'i0', 'p0', 
      
   
What would you recommend in simulating a larger transformer?

**Comment by TonyXiang8787** [comment, marked as answer]
Hi @willemijnbrus,

When you scale up a transformer rated power `sn`, you have to also scale up linearly for the attribute `pk` and `p0`. The rest you don't have to change them. They are relative values.

**Reply by willemijnbrus** [reply]
Thank you Tony!
